pkgs/development/python-modules/mautrix/default.nix

```diff
@@ -4,11 +4,11 @@
 
 buildPythonPackage rec {
   pname = "mautrix";
-  version = "0.14.10";
+  version = "0.15.6";
```

Comment:
Isn't this update breaking and requires all other mautrix things to be updated at the same time?

Reply:
Yes, see the discussion here: https://github.com/NixOS/nixpkgs/pull/166449